docs/developer/other/RELEASE.md

```diff
@@ -1,28 +1,126 @@
+    - This is granted via a Jira ticket from a uPortal committer
+3. [Set up public PGP key on a server](https://central.sonatype.org/pages/working-with-pgp-signatures.html)
+    - Generate a key pair `gpg2 --gen-key`
+    - If you choose to have an expiration date, edit the key via `gpg2 --edit-key {key ID}`
+    - Determine the key ID and keyring file `gpg2 --list-keys` (the key ID is the `pub` ID)
+    - Distribute your public key `gpg2 --keyserver hkp://pool.sks-keyservers.net --send-keys {key ID}`
+
 ## Setup
 
-in `$HOME/.gradle/gradle.properties` your credentials for Sonatype OSS
-Repository Hosting and your configuration information for signing artifacts with
-GNU Privacy Guard (GnuPG).
-([details](https://docs.gradle.org/current/userguide/signing_plugin.html#sec:signatory_credentials)).
+Export your secret keyring via `gpg2 --keyring secring.gpg --export-secret-keys > ~/.gnupg/secring.gpg`
+
+In `$HOME/.gradle/gradle.properties` place your credentials for the Sonatype OSS Repository Hosting and your configuration information for signing artifacts with GNU Privacy Guard (GnuPG).  Use the key ID from the PGP prerequisite and the keyring file. ([details](https://docs.gradle.org/current/userguide/signing_plugin.html#sec:signatory_credentials)).
 
 ```properties
 ossrhUsername={username}
 ossrhPassword={secret}
 
-signing.keyId=24875D73
+signing.keyId={key ID}
 signing.password={secret}
-signing.secretKeyRingFile=/Users/me/.gnupg/secring.gpg
+signing.secretKeyRingFile={keyring file}
 ```
+Setup is only required to be done once.
+
+## Which Repo?
 
-## Running a release
+We encourage performing releases directly from a clone of the official repository rather than a fork to avoid extra steps.
 
-Run the following command:
+This means when testing on `uPortal-start` for the release, you should use the `apereo` repository but configure the `uPortalVersion` to be the SNAPSHOT version that you'll build in the following steps.
 
+## Testing
+
+Build a clean version of `uPortal-start` with the quickstart data set and perform at least light testing, especially around features that have been fixed or enhanced.
+
+### Prepare For Testing
+
+Build uPortal locally:
+```bash
+~.../apereo/uPortal$ ./gradlew clean install
+```
+
+Point your uPortal-start to the local uPortal build:
+Using the uPortal version, `.../apereo/uPortal/gradle.properties` > `version=XYZ` , add it into `.../apereo/uPortal-start/gradle.properties` > `uPortalVersion=XYZ`
+
+Run uPortal-start with the local build
 ```sh
-./gradlew clean release --no-parallel
+~.../apereo/uPortal-start$ ./gradlew clean portalInit
 ```
 
-:warning: During the `release` task, you will be prompted for a release version
-(e.g. `5.0.3`) and a new version number for the branch (e.g. `5.0.4-SNAPSHOT`)
+### Verify
+
+* Unit tests are automatically run on commits, so ensure the latest commit's CI build passed.
+* FUTURE - Need to run the cross browser platform tests and the performance tests
```

Comment:
Is there an issue tracking this?

Reply:
@cbeach47 Did you add an issue for this?